pretyping/evarconv.ml

```diff
@@ -305,8 +305,15 @@ let check_conv_record env sigma (t1,sk1) (t2,sk2) =
   let params1, c1, extra_args1 =
     match arg with
     | Some c -> (* A primitive projection applied to c *)
+      let meta_type mv = match metas with
+      | None -> None
+      | Some metas ->
+        match Metamap.find mv metas with
+        | Cltyp (_, b) -> Some b.Evd.rebus
+        | Clval (_, _, b) -> Some b.Evd.rebus
+      in
```

Comment:
I'd rather move this to the caller of check_conv_record

Reply:
This is also part of the next PR.